config/management/src/smoke_test.rs

```diff
@@ -120,6 +120,23 @@ fn smoke_test() {
     for (i, mut config) in configs.iter_mut().enumerate() {
         let ns = i.to_string();
         let waypoint = helper.create_waypoint(&ns).unwrap();
+
+        // save the waypoint into the common namespace so that validators can
+        // read it and perform insert-waypoint
+        common_storage
+            .set(
+                crate::constants::WAYPOINT,
+                Value::String(waypoint.to_string()),
+            )
+            .unwrap();
+
+        // TODO(Rustinante): figure out how to run the validator
+        if let Some(_validator_config) = &config.validator_network {
+            helper
+                .insert_waypoint(crate::constants::COMMON_NS, Some(&waypoint.to_string()))
```

Comment:
insert should happen within the current nodes ns and it should be read from the remote secure storage (ns) that was create in the create-waypoint